channels.yaml

```diff
@@ -2,13 +2,13 @@ releases:
   - version: v1.18.20+k3s1
     minChannelServerVersion: v2.6.0-alpha1
     maxChannelServerVersion: v2.6.99
-  - version: v1.19.15+k3s2
+  - version: v1.19.16+k3s1
     minChannelServerVersion: v2.6.0-alpha1
     maxChannelServerVersion: v2.6.99
-  - version: v1.20.11+k3s2
+  - version: v1.20.12+k3s1
     minChannelServerVersion: v2.6.0-alpha1
     maxChannelServerVersion: v2.6.99
-  - version: v1.21.4+k3s1
+  - version: v1.21.6+k3s1
```

Comment:
Can't bump entries if they have serverArgs/agentArgs/charts. Needs a new entry at the bottom of the list.